product_docs/docs/eprs/6.2/03_installation/02_installing_from_cli.mdx

```diff
@@ -156,3 +156,10 @@ The following is a list of the parameters that may be specified when running the
 > The password for the operating system account. There is no default for this parameter.
 
 Successful installation of xDB Replication Server results in the creation of directory structures and files in your host environment as described in Section [Post-Installation Host Environment](05_post_installation_host_environment/#post_installation_host_environment).
+
+Before using Replication Server, you must download and install JDBC drivers. See [Installing a JDBC driver](installing_jdbc_driver) for details.
+
+!!! Note
```

Comment:
Similar to the comment I added in the stackbuilder installation page, I wonder if this note is applicable in when installing using the interactive installer via the command line.  If it is, then this note should be moved earlier in the page where the JDK installation is mentioned (in a note near the top of the page)